dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild

```diff
@@ -0,0 +1,287 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit flag-o-matic eutils autotools multilib user readme.gentoo versionator
+
+MY_P=${PN/f/F}-$(replace_version_separator 4 -)
+#MY_P=${PN/f/F}-${PV/_rc/-ReleaseCandidate}
+
+DESCRIPTION="A relational database offering many ANSI SQL:2003 and some SQL:2008 features"
+HOMEPAGE="http://www.firebirdsql.org/"
+SRC_URI="
+	mirror://sourceforge/firebird/${MY_P}.tar.bz2
+	 doc? (	ftp://ftpc.inprise.com/pub/interbase/techpubs/ib_b60_doc.zip )"
+
+LICENSE="IDPL Interbase-1.0"
+SLOT="0"
+KEYWORDS="~amd64 -ia64 ~x86"
```

Comment:
23 Mar 2006; Aron Griffis <agriffis@gentoo.org> firebird-1.5.1.ebuild,
509	  firebird-1.5.2.ebuild, firebird-1.5.3.ebuild, firebird-1.5.3-r1.ebuild:
510	  No versions of firebird presently build on ia64. 1.5.1 attempts to build
511	  with x86-only CFLAGS and 1.5.3 doesn't have IMPLEMENTATION #defined for
512	  ia64. Mark them all -ia64 (with use-masking to following)